packages/mobile/src/screens/sign-on-screen/components/layout.tsx

```diff
@@ -47,6 +47,8 @@ export const Page = (props: PageProps) => {
       style={[
         css({
           zIndex: 1,
+          minHeight:
```

Comment:
From what I can tell, Android seems to shrink `height: 100%` with the keyboard, whereas iOS doesnt?